commons/src/main/java/org/openmrs/eip/app/management/config/ManagementDataSourceConfig.java

```diff
@@ -88,8 +99,19 @@ public SpringLiquibase getSpringLiquibaseForMgtDB(@Qualifier("mngtDataSource") D
 		liquibase.setDatabaseChangeLogTable("LIQUIBASECHANGELOG");
 		liquibase.setDatabaseChangeLogLockTable("LIQUIBASECHANGELOGLOCK");
 		liquibase.setShouldRun(true);
+		releaseLiquibaseLock(dataSource);
 		
 		return liquibase;
 	}
-	
+
+	public void releaseLiquibaseLock(DataSource dataSource) {
+		try {
+			Database database = DatabaseFactory.getInstance()
+					.findCorrectDatabaseImplementation(new JdbcConnection(dataSource.getConnection()));
+			LockServiceFactory.getInstance().getLockService(database).forceReleaseLock();
+		} catch (DatabaseException | SQLException | LockException e) {
+			log.error("Error occurred while liquibase forceReleaseLock {}", e.getMessage(), e);
+			throw new RuntimeException(e);
```

Comment:
I'm not certain that a failure here should throw a `RuntimeException`. This code is essentially just a work-around for an occasional failure state and we don't necessarily want to add a new failure state _just_ because something went wrong in the unlocking code. I think it's better to log the error and let the program continue. It may fail down the line, but at least we won't introduce new failures.

Reply:
Sure makes sense.